src/algorithms/telemetry_decoder/gnuradio_blocks/sbas_l1_telemetry_decoder_gs.cc

```diff
@@ -415,7 +465,6 @@ int sbas_l1_telemetry_decoder_gs::general_work(int noutput_items __attribute__((
     current_symbol = in[0];
     // copy correlation samples into samples vector
     d_sample_buf.push_back(current_symbol.Prompt_I);  // add new symbol to the symbol queue
-
```

Comment:
don't hate the vertical whitespace. In particular, blank lines before comments facilitate readability. Beware of gratuitously changing things to match **your** IDE; your change may not aid readability on someone else's tool.